blockmanager.go

```diff
@@ -63,7 +63,7 @@ type invMsg struct {
 // headersMsg packages a bitcoin headers message and the peer it came from
 // together so the block handler has access to that information.
 type headersMsg struct {
-	headers *wire.MsgHeaders
+	headers []*wire.BlockHeader
```

Comment:
I don't think we need this change, as we don't need to funnel this through the block manager. 

This is the interface between the p2p network and our internal header/block storage. For the side loader, we don't need the p2p side, so we can insert it directly into the datababase, using a refactored function from the header validation along the way.